# flake8: noqa F401         - import not used error
from .s3tasks import upload_object, sanitise_bucket_name, S3Tasks

name = 's3ops'
